# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/cranelift/baldrapi.h
# Commit: 1ab8383d17ee
# Full Hash: 1ab8383d17ee1e5db164339236d64bcb7168b225
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Atomic-rmw baseline operations. r=yury
#   
#   Atomic ADD, SUB, AND, OR, XOR, XCHG, CMPXCHG, as well as WAIT and
#   NOTIFY for the baseline compiler.
#   
# ==============================================================================

diff -r 6089b4360edc -r 1ab8383d17ee js/src/wasm/cranelift/baldrapi.h
--- a/js/src/wasm/cranelift/baldrapi.h	Mon Oct 18 07:31:55 2021 +0000
+++ b/js/src/wasm/cranelift/baldrapi.h	Mon Oct 18 07:31:56 2021 +0000
@@ -209,9 +209,9 @@
   TruncF64,
   PreBarrier,
   PostBarrier,
-  WaitI32,
-  WaitI64,
-  Wake,
+  WaitI32, /* WaitI32M32 */
+  WaitI64, /* WaitI64M32 */
+  Wake,    /* WakeM32 */
   Limit
 };
 
